bigtop-tests/test-artifacts/hbase/src/main/groovy/org/apache/bigtop/itest/hbase/smoke/TestImportTsv.groovy

```diff
@@ -89,7 +92,8 @@ public class TestImportTsv {
     // delete data and junk from HDFS
     sh.exec("hadoop fs -rmr -skipTrash $DATADIR1",
       "hadoop fs -rmr -skipTrash $DATADIR2",
-      "hadoop fs -rmr -skipTrash /user/$USER/partitions_*");
+      "hadoop fs -rmr -skipTrash /user/$USER/hbase*",
```

Comment:
No prob. I see other JIRAs addressing the DEPRECATED issue of rmr, hence I think it's a good time to address it here as well.